mailbox/store/src/main/java/org/apache/james/mailbox/store/mail/MessageIdMapper.java

```diff
@@ -26,11 +26,10 @@
 import org.apache.james.mailbox.model.MessageId;
 import org.apache.james.mailbox.store.mail.MessageMapper.FetchType;
 import org.apache.james.mailbox.store.mail.model.MailboxMessage;
-import org.apache.james.mailbox.store.mail.model.Message;
 
 public interface MessageIdMapper {
 
-    List<Message> find(List<MessageId> messageIds, FetchType fetchType);
+    List<MailboxMessage> find(List<MessageId> messageIds, FetchType fetchType);
```

Comment:
Well why ?

 - The implementation basically returns MailboxMessage
 - We return messages that are in the different mailboxes. Hence it makes sens to have mailbox information returned.
 - Even more that we need it for the Message results of the manager.

Here is my question : Why loosing information when we need it ?